aiocache/backends/memcached.py

```diff
@@ -10,7 +10,7 @@ def __init__(self, endpoint="127.0.0.1", port=11211, pool_size=2, loop=None, **k
         super().__init__(**kwargs)
         self.endpoint = endpoint
         self.port = port
-        self.pool_size = pool_size
+        self.pool_size = int(pool_size)
```

Comment:
unrelated with PR but querystring from `from_url` method always parses args as strings. This is helpful anyway as a standalone feature